docs/Device implementation tutorial.md

```diff
@@ -42,7 +42,7 @@ As per the [MS-05-02](https://specs.amwa.tv/ms-05-02/branches/v1.0-dev/docs/Bloc
 
 All devices have at the very least a `root block` which is the top most block in the device tree. The root block has an `oid` of 1 and the role of `root`.
 
-Control classes which are nested inside a block are advertised using descriptors in the members property (`2p10`) of [NcBlock](https://specs.amwa.tv/ms-05-02/branches/v1.0-dev/idl/NC-Framework.html).
+Control classes which are nested inside a block are advertised using descriptors in the members property of [NcBlock](https://specs.amwa.tv/ms-05-02/branches/v1.0-dev/docs/Framework.html#ncblock).
```

Comment:
Backticks around `members` to be consistent with other documents in this INFO spec (e.g. [How To practical examples](https://specs.amwa.tv/info-006/branches/main/docs/How_To_practical_examples.html)) 
```suggestion
Control classes which are nested inside a block are advertised using descriptors in the `members` property of [NcBlock](https://specs.amwa.tv/ms-05-02/branches/v1.0-dev/docs/Framework.html#ncblock).
```